### DISCUSSION TITLE
IntelliSense client not available, using Tag Parser for go to definition.

### DISCUSSION THREAD

**Discussion by Xubaidu** [microsoft/vscode-cpptools#8711] [Q&A]
why does IntelliSense client not available? is there anything wrong with my network?

**Comment by Colengms** [comment]
Hi @Xubaidu .  There are a few possible reasons for this error message.  Could you open an issue in this repo with your specific repro steps?  The new issue form should request information such as your settings and build configuration which may help us identify why this is failing.
